src/Tutor.Core/DomainModel/Notes/NoteService.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using FluentResults;
+
+namespace Tutor.Core.DomainModel.Notes
+{
+    public class NoteService : INoteService
+    {
+        private readonly INoteRepository _noteRepository;
+
+        public NoteService(INoteRepository noteRepository)
+        {
+            _noteRepository = noteRepository;
+        }
+
+        public Result<Note> Save(Note note)
+        {
+            return Result.Ok(_noteRepository.Save(note));
+        }
+
+        public Result<int> Delete(int id)
+        {
+            var note = _noteRepository.FindById(id);
```

Comment:
If this is the only use for FindById why not integrate it into the delete method and remove it from the repo interface?